microcosm_flask/routing.py

```diff
@@ -21,6 +21,7 @@ def make_path(graph, path):
     enable_audit=True,
     enable_basic_auth=False,
     enable_cors=True,
+    enable_metrics=False,
```

Comment:
We can enable metrics globally. By default, we do not because it will attempt to resolve the binding key `metrics_timing` and will fail